product/views.py

```diff
@@ -58,3 +72,119 @@ def get(self, request, product_id):
                 'articleProductDetailImg'      : [image.image_url for image in product.productimageurl_set.all()]
             }
         }, status=200)
+
+class CategoryView(View):
+    def get(self, request):
+        try:
+            first_categories = FirstCategory.objects.prefetch_related('secondcategory_set__thirdcategory_set')
+            print (first_categories)
+            categories = [{
+                'id'   : first_category.id,
+                'name' : first_category.name,
+                'second_category' : [{
+                    'id'   : second_category.id,
+                    'name' : second_category.name,
+                    'third_category': [{
+                        'id'   : third_category.id,
+                        'name' : third_category.name,
+                    }for third_category in second_category.thirdcategory_set.all()]
+                }for second_category in first_category.secondcategory_set.all()]
+            }for first_category in first_categories]
+
+            return JsonResponse({ 'message:':'SUCCESS', 'categories':categories}, status=200)
```

Comment:
```suggestion
            return JsonResponse({'categories':categories}, status=200)
```
성공했다면 성공메세지는 따로 주지않아도 괜찮습니다.

Reply:
수정하였습니다